server/storage/sql_models.go

```diff
@@ -11,6 +11,16 @@ const (
 	changeCategoryDeletion = "deletion"
 )
 
+// TUFFileTableName returns the name used for the tuf file table
+var TUFFileTableName = func() string {
```

Comment:
Then again, they should also NEVER be used in more than one context (i.e. through the appropriate struct) so I actually prefer them being defined directly in the struct methods. It tightly binds them to the correct context.